cmd/main.go

```diff
@@ -129,7 +129,11 @@ func main() {
 		NSXConfig: cf,
 	}
 
-	// Start the security policy controller.
+	if cf.CoeConfig.EnableVPCNetwork && commonService.NSXClient.NSXCheckVersionFor(nsx.FeatureVPC) {
+		log.V(1).Info("VPC mode enabled")
+		// Start controllers which only supports VPC
+	}
+	// Start the security policy controller, it supports VPC and non VPC mode
 	StartSecurityPolicyController(mgr, commonService)
 	// Start the NSXServiceAccount controller.
 	if cf.EnableAntreaNSXInterworking {
```

Comment:
StartNSXServiceAccountController might be used in non-vpc mode as well, need to confirm with GuRan.

Reply:
ditto.